src/Web/Views/Shared/_Layout.cshtml

```diff
@@ -60,19 +62,26 @@
     <header class="navbar navbar-light navbar-static-top">
         <div class="container">
             <article class="row">
-                <section class="col-lg-6 col-md-5 col-xs-11">
+                <section class="col-lg-2 col-md-4 col-xs-11">
                     <a asp-area="" asp-controller="Catalog" asp-action="Index" class="navbar-brand">
                         <img src="~/images/brand.png" alt="Microsoft Fraud Protection Merchant Application" title="@apiBaseUrl" />
                     </a>
                 </section>
                 <partial name="_LoginPartial" />
+                @using (Html.BeginForm("Update", "Environment", FormMethod.Post, new { id = "environmentupdate" }))    
```

Comment:
When I register on the site with an average length email address the email and cart overlap now:
![image](https://user-images.githubusercontent.com/165767/165154336-b5dd16c6-14f1-4798-b520-c28e7f5a2c2f.png)

Reply:
![image](https://user-images.githubusercontent.com/544943/165186117-3178a70f-3271-44c5-9d06-52c1b47a40f8.png)

![image](https://user-images.githubusercontent.com/544943/165186173-e4569c14-c23b-4e84-8663-6068f231f87a.png)